src/commands.rs

```diff
@@ -101,36 +111,97 @@ impl Formatter for Screens {
     }
 }
 
+impl Assets {
+    pub fn new(value: serde_json::Value) -> Self {
+        Self { value }
```

Comment:
Isn't it what default constructor does?:
```
let asset: Assets = Assets{value: value};
```

Reply:
No. Constructing with `Assets {value: value}` requires you to know about the object structure while when constructing from new all the magic happens inside new. Encapsulation.